src/main/java/org/dependencytrack/model/scheduled/policyviolations/PolicyViolationSummary.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.dependencytrack.model.scheduled.policyviolations;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.dependencytrack.model.PolicyViolation;
+import org.dependencytrack.model.Project;
+
+/* 
+ * Part of the ScheduledPolicyViolationsIdentified Template Models.
+ * Contains packed summarized informations about the identified policy violations, grouped by the affected projects.
+ */
+public final class PolicyViolationSummary {
```

Comment:
Consider making this a [`record`](https://docs.oracle.com/en/java/javase/21/language/records.html) instead.